src/measure_theory/lp_space.lean

```diff
@@ -1449,6 +1449,14 @@ begin
   { exact norm_indicator_const_Lp hp_pos hp_top, },
 end
 
+@[simp] lemma indicator_const_empty :
+  indicator_const_Lp p measurable_set.empty (by simp : μ ∅ ≠ ∞) c = 0 :=
+begin
+  rw Lp.eq_zero_iff_ae_eq_zero,
```

Comment:
It does not change much, but you could also have used ext:
```suggestion
  ext1,
```

Reply:
When I try this, I hit the problem that `⇑0` is only ae-eq to 0, so I need to add in a use of the lemma `Lp.coe_fn_zero`.
```lean
  ext1,
  refine (indicator_const_Lp_coe_fn.trans _).trans (Lp.coe_fn_zero _ _ _).symm,
  simp [set.indicator_empty'],
```